__tests__/new/sendTransaction.test.js

```diff
@@ -91,6 +91,7 @@ test('prepareTxData', async () => {
         authorities: 0,
       },
       'an-input-from-storage',
+      'an-input-from-storage',
```

Comment:
This extra input and output are because we are calling `fillTx` twice, one for HTR and another for the `01` custom token.
Before we would get only 1 since we made only 1 call to `fillTx`.